l5kit/l5kit/dataset/select_agents.py

```diff
@@ -95,7 +82,7 @@ def get_valid_agents(
     agents = dataset.agents[agents_range_start:agents_range_end]
 
     agents_dict = defaultdict(list)
-    agents_selected_mask = np.zeros(len(agents), dtype=np.bool)
+    agents_mask = np.zeros((len(agents), 2), dtype=np.int)
```

Comment:
Can we have a comment here like in the PR description telling what agents_mask means? (available_frame_past, available_frame_futures).
In general, I have to say, I did not find this so easy to follow.